onnxruntime/core/providers/cuda/object_detection/non_max_suppression.cc

```diff
@@ -66,7 +66,9 @@ Status NonMaxSuppression::ComputeInternal(OpKernelContext* ctx) const {
           GetCenterPointBox(),
           batch_index,
           class_index,
-          max_output_boxes_per_class,
+          max_output_boxes_per_class > std::numeric_limits<int>::max()
```

Comment:
>max_output_boxes_per_class [](start = 10, length = 26)

this is in the for loop, why do not you move this out of the for loop and only do it once?